osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
Our sensor models require counter clockwise ordering, that's why I suggested it. 
I guess in the end the sensor model has to check the order anyway and potentially correct it if the wrong order was provided. So I would keep the requirement for CCW ordering in the interface if no one objects, but keep it up to the model to really check if this requirement was fulfilled.
I agree with the two reference frames mentioned in #39, ground truth should be in world coordinates and the sensed contour in sensor coordinates. We do need access to the specifications where the sensor is mounted to do further transformations ourselves. I guess this is given in relation to the middle of the bounding box? Then we also need the relations between the middle of the bounding box and the middle of the rear/front axis since we have other coordinate systems which are in relation to these points. 
@nHirsenkorn These specs are not yet implemented in OSI, are they?